src/printer-htmlparser2.js

```diff
@@ -103,7 +111,15 @@ function printAttributes(attribs) {
         if (attribs[name] === "") {
           return name;
         }
-        return concat([name, '="', attribs[name], '"']);
+
+        // FastPath requires the next node to be a property of the current one
+        node._currentAttribute = {
```

Comment:
This is the least worst way of doing this that I could think of.

The _correct_ way would be to reshape the AST between parsing and printing, but I'm not sure if that would hurt performance?